OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/SegmentAnalytics.java

```diff
@@ -1097,4 +1098,23 @@ public void trackProfileScreenEvent(@NonNull String eventName, @NonNull String b
         aEvent.properties.putValue(Keys.NAME, biValue);
         trackSegmentEvent(eventName, aEvent.properties);
     }
+
+    @Override
+    public void trackVideoDownloadQualityClicked(String eventName, String biValue, String courseId) {
+        final SegmentEvent aEvent = new SegmentEvent();
+        aEvent.properties.putValue(Keys.NAME, biValue);
+        if (!TextUtils.isEmpty(courseId)) {
+            aEvent.properties.put(Keys.COURSE_ID, courseId);
+        }
+        trackSegmentEvent(eventName, aEvent.properties);
+    }
+
+    @Override
+    public void trackVideoDownloadQualityChanged(@NonNull VideoQuality selectedVideoQuality, @NonNull VideoQuality oldVideoQuality) {
+        final SegmentEvent aEvent = new SegmentEvent();
+        aEvent.properties.putValue(Keys.NAME, Analytics.Values.VIDEO_DOWNLOAD_QUALITY_CHANGED);
+        aEvent.data.putValue(Keys.VALUE, selectedVideoQuality.name().toLowerCase().replaceAll("option_", ""));
+        aEvent.data.putValue(Keys.OLD_VALUE, oldVideoQuality.name().toLowerCase().replaceAll("option_", ""));
+        trackSegmentEvent(Analytics.Screens.VIDEO_DOWNLOAD_QUALITY_CHANGED, aEvent.properties);
```

Comment:
Can we use Analytics.Event value instead of Analytics.Screen.

Reply:
done.